Source/ORTS.Common/SystemInfo.cs

```diff
@@ -200,7 +200,7 @@ static string WriteInstalledRuntimes(TextWriter output, string versionKeyName, R
 
         static void WriteGraphicsAdapter(TextWriter output)
         {
-            foreach (var adapter in GraphicsAdapter.Adapters)
+ /*           foreach (var adapter in GraphicsAdapter.Adapters)
```

Comment:
removed

Reply:
Removed

Reply:
It looks like we can still list the device name, description, plus the device and vendor IDs. This may still be useful when debugging from someone's log file.